lib/rust/frp/src/network.rs

```diff
@@ -99,25 +103,35 @@ impl Network {
     /// network, like animation instances.
     pub fn store<T: 'static + CloneRef>(&self, item: &T) {
         let item = item.clone_ref();
-        self.data.storage.borrow_mut().push(Box::new(item));
+        self.store_boxed(Box::new(item));
+    }
+
+    /// Store an arbitrary boxed item.
+    #[inline(never)]
+    fn store_boxed(&self, item: Box<dyn Any>) {
+        self.data.storage.borrow_mut().push(item);
     }
 
     /// Register the node and return it's weak reference.
     pub fn register_raw<T: HasOutputStatic>(&self, node: stream::Node<T>) -> stream::WeakNode<T> {
         let weak = node.downgrade();
-        let node = Box::new(node);
-        self.data.nodes.borrow_mut().push(node);
+        self.register_boxed(Box::new(node));
         weak
     }
 
     /// Register the node and return a new `Stream` reference.
     pub fn register<Def: HasOutputStatic>(&self, node: stream::Node<Def>) -> Stream<Output<Def>> {
         let stream = node.clone_ref().into();
-        let node = Box::new(node);
-        self.data.nodes.borrow_mut().push(node);
+        self.register_boxed(Box::new(node));
         stream
     }
 
+    /// Register a boxed node. Non-generic part of registration.
+    #[inline(never)]
```

Comment:
Why do we need this attribute? Add a comment with explanation.

Reply:
Added explanation in the comment.